src/providers/wcs/qgswcscapabilities.h

```diff
@@ -34,24 +34,59 @@
 class QNetworkAccessManager;
 class QNetworkReply;
 
-//! CoverageSummary structure
+/**
+ * \brief Metadata link property structure.
+ *
+ *  Gets the optional medatadaLink element
```

Comment:
```suggestion
 *  Contains the optional medatadaLink element
```

It's just a data structure and doesn't "get" anything :)